app/components/pokemon-item/pokemon-item.jsx

```diff
@@ -8,19 +8,23 @@ export const PokemonItem = ({ pokemon }) => {
 
   return (
     <div className={styles['pokemon-item']}>
-      <Link href={`/${pokemon.name}`}>
+      <Link href={`/${pokemon?.name ?? 'unknown'}`}>
```

Comment:
This shouldn't be set to unknown, as it will take you to {url}/unknown when you click on it. Leave it as pokemon?.name. Is there a case where name is not defined?